deno/payloads/v10/_interactions/base.ts

```diff
@@ -31,21 +38,48 @@ export interface APIMessageInteractionMetadata {
 	 */
 	user: APIUser;
 	/**
-	 * IDs for installation context(s) related to an interaction. Details in Authorizing Integration Owners Object
+	 * IDs for installation context(s) related to an interaction
 	 */
 	authorizing_integration_owners: APIAuthorizingIntegrationOwnersMap;
 	/**
 	 * ID of the original response message, present only on follow-up messages
 	 */
 	original_response_message_id?: Snowflake;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/message#message-interaction-metadata-object-application-command-interaction-metadata-structure
+ */
+export interface APIApplicationCommandInteractionMetadata extends APIBaseInteractionMetadata {
```

Comment:
These interfaces need to define the specific type they point to so you can properly deduce it from the tuple